docs/sources/reference/share_panel.md

```diff
@@ -1,43 +1,17 @@
 +++
-title = "Sharing"
+title = "Sharing a Panel"
 keywords = ["grafana", "dashboard", "documentation", "sharing"]
 type = "docs"
 [menu.docs]
 parent = "dashboard_features"
 weight = 6
 +++
 
-# Sharing features
+# Share Panel
```

Comment:
```suggestion
# Share panel
```

From the [style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md): 
> Write headings in sentence case, not title case.